searchconsole/query.py

```diff
@@ -178,6 +178,32 @@ def search_type(self, search_type):
 
         return self
 
+    @utils.immutable
+    def data_state(self, data_state):
+        """
+        Return a new query filtered by the specified dataState, which allows you 
+        to include fresh (not finalized) data in your API call.  
+
+        Fresh data: data as recent as less than a day old. Fresh data point can 
+        be replaced with the final data point after a few days. 
+
+        Args:
+            dataState (str): The dataState you would like to use for your report. 
```

Comment:
This parameter is now called `data_state`.

Reply:
@joshcarty: *facepalm* 

Done :)